demisto_sdk/__main__.py

```
@@ -923,7 +923,15 @@ def secrets(config, **kwargs):
     default="from-yml",
     help="The docker image to check package on. Possible values: 'native:maintenance', 'native:ga', 'native:dev',"
     " 'all', a specific docker image from Docker Hub (e.g devdemisto/python3:3.10.9.12345) or the default"
-    " 'from-yml'.",
+    " 'from-yml', 'native:target'. To run lint only on native supported content with a specific image,"
+    " use 'native:target' with --docker-image-target <specific-image>.",
+)
+@click.option(
+    "-dit",
+    "--docker-image-target",
+    default="",
+    help="The docker image to lint native supported content with. Should only be used with "
+    "--docker-image native:target. Will be ignored otherwise.",
```

let's raise an error when used without, instead of ignoring